index.html

```diff
@@ -17,11 +17,26 @@
 </head>
 <body>
   <header>
-    <div class="container">
-    </div>
+    <p id="header-p" class="header-P">Code Journal</p>
   </header>
   <main class="container">
-    <div data-view="entry-form"></div>
+    <h1>New Entry</h1>
+    <div class="container">
+      <input type="image" src="images/placeholder-image-square.jpg" class="url-pic">
+    </div>
```

Comment:
You should be utilizing rows & columns here. Below is a snippet from my project. Notice how the `row` div isn't closed off in this snippet. There's some more columns that are inside of it. Basically this `row` should hold the image and all of your form elements
```
<div class="row">
  <div class="photo-wrapper column-half">
    <img id="entry-image" src="images/placeholder-image-square.jpg" alt="Placeholder image">
  </div>
  ```